distribution_test.go

```diff
@@ -134,6 +134,38 @@ func TestBundleDownloadUsingSpec(t *testing.T) {
 	cleanDistributionTest(t)
 }
 
+func TestBundleCreateByAql(t *testing.T) {
+	initDistributionTest(t)
+
+	// Upload files
+	specFile, err := tests.CreateSpec(tests.DistributionUploadSpecB)
+	assert.NoError(t, err)
+	err = artifactoryCli.Exec("u", "--spec="+specFile)
+	assert.NoError(t, err)
+
+	// Create release bundle by AQL
+	spec, err := tests.CreateSpec(tests.DistributionCreateByAql)
+	assert.NoError(t, err)
+	err = distributionCli.Exec("rbc", tests.BundleName, bundleVersion, "--spec="+spec, "--sign")
+	assert.NoError(t, err)
```

Comment:
Let's add this assertion to all the other (older) usages of *distributionCli.Exec* in this file.